misc/incremental_checker.py

```diff
@@ -48,49 +61,56 @@ def execute(command: List[str], fail_on_error: bool = True) -> Tuple[str, str, i
     stdout_bytes, stderr_bytes = proc.communicate()  # type: Tuple[bytes, bytes]
     stdout, stderr = stdout_bytes.decode('utf-8'), stderr_bytes.decode('utf-8')
     if fail_on_error and proc.returncode != 0:
-        print("RETURN CODE:", proc.returncode)
+        print('EXECUTED COMMAND:', ' '.join(command))
```

Comment:
On the off-chance that there might be whitespace in some of the arguments, maybe just print the repr() of the list?